server/selinux/server/pulp-celery.te

```diff
@@ -158,10 +161,23 @@ optional_policy(`
 
 ######################################
 #
-# The rsync distributors need to be able to execute rsync.
+# Adds a SELinux boolean called pulp_manage_rsync that enables Pulp to execute rsync and ssh.
+# Defaults to false.
 #
 
-rsync_exec(celery_t)
+gen_tunable(pulp_manage_rsync, false)
+
+optional_policy(`
+    tunable_policy(`pulp_manage_rsync', `
+        # Pulp executes rsync over ssh. Pulp needs to write to /var directories. Allow creation of
+        # a socket file. Allow managing a socket.
+        rsync_exec(celery_t)
+        ssh_exec(celery_t)
```

Comment:
Rsync distributors run rsync over ssh to transfer files to a remote server [0]. 

[0] https://github.com/pulp/pulp/blob/master/server/pulp/plugins/rsync/publish.py#L101-L108